ci/infrastructure/scripts/infrastructure.sh

```
@@ -1110,7 +1113,7 @@ createBuildReport() {
     echo "$VS_HOST_IP_ADDRESS" > env_host.txt
   else
     EXIT_CODE=127
-    VS_MAIL_NOTIFY_BUILD_SUBJECT="environment build FAILED for $JOB_BASE_NAME in $VS_PARENT_JOB_NAME"
+    VS_MAIL_NOTIFY_BUILD_SUBJECT="environment build FAILED for $JOB_BASE_NAME in $VS_PARENT_JOB_NAME_FULL"
     echo "# " | tee $VS_MAIL_NOTIFY_BUILD_MESSAGE
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)